### PR TITLE
Regex - Extract all occurence of COVID-19 #2

### PR DESCRIPTION
**Pull request by armaf002**
This introduces a Python script named "task.py" that utilizes a regular expression (regex) pattern to extract all mentions of the keyword "COVID-19" from a provided text block, regardless of letter casing.

### CODE REVIEW

**Review by elseagle — APPROVED** [review]
LGTM! Good Job

**Comment by hammedb197** [comment]
@armaf002 can you move your work folder into Regex/

**Comment by armaf002** [comment]
@hammedb197 Alright, I'll do that

**Comment by armaf002** [comment]
> @armaf002 can you move your work folder into Regex/

@hammedb197 I've moved the folder into Regex/

File: Regex/Extract all occurence of COVID-19 #2/README.md
Comment (elseagle) [inline Regex/Extract all occurence of COVID-19 #2/README.md:1]: your folder should not have space in it. Use under score and ensure they are lowercase

File: ridwan_regex_task/extract_covid19.py
Comment (elseagle) [inline ridwan_regex_task/extract_covid19.py:1]: This folder and file should be inside the Regex folder else delete it if not needed

**Review by elseagle — CHANGES_REQUESTED** [review]

File: Regex/Extract all occurence of COVID-19 #2/README.md
Comment (armaf002) [reply Regex/Extract all occurence of COVID-19 #2/README.md:1]: That was my mistake, I'll rectify it

File: ridwan_regex_task/extract_covid19.py
Comment (armaf002) [reply ridwan_regex_task/extract_covid19.py:1]: I'll delete all the irrelevant files and folders

Comment (armaf002) [reply ridwan_regex_task/extract_covid19.py:1]: All the not needed files has been deleted

File: Regex/Extract all occurence of COVID-19 #2/README.md
Comment (armaf002) [reply Regex/Extract all occurence of COVID-19 #2/README.md:1]: It has been rectified

**Pull request merged**